sw/c/common/uart.c

```diff
@@ -7,11 +7,23 @@
 #include "sonata_system.h"
 #include "dev_access.h"
 
+#define CPU_FREQ_HZ (  50000000)
```

Comment:
Can't we use system clock frequency defined in the sonata system header here?

Reply:
Indeed. I overlooked that when modifying uart.c to drive the OT UART.